shared/django_apps/user_measurements/partitioning.py

```diff
@@ -0,0 +1,27 @@
+from dateutil.relativedelta import relativedelta
+from psqlextra.partitioning import (
+    PostgresCurrentTimePartitioningStrategy,
+    PostgresPartitioningManager,
+    PostgresTimePartitionSize,
+)
+from psqlextra.partitioning.config import PostgresPartitioningConfig
+
+from shared.django_apps.user_measurements.models import UserMeasurement
+
+# Overlapping partitions will cause errors - https://www.postgresql.org/docs/current/ddl-partitioning.html#DDL-PARTITIONING-DECLARATIVE -> "create partitions"
+manager = PostgresPartitioningManager(
```

Comment:
does this specific manager object get used anywhere

Reply:
Yeap, this is the code that enables running the `python manage.py pgpartition` command, which automatically creates the partitions specified in these settings. I followed the config instructions of the docs, https://django-postgres-extra.readthedocs.io/en/latest/table_partitioning.html#configuration

I also centralized the logic here in case the partition logic wants to be called in any of its child repositories